tools/android/MASTG-TOOL-0030.md

```diff
@@ -17,4 +17,4 @@ pip install angr
 
 Comprehensive documentation, including an installation guide, tutorials, and usage examples are available on [Angr's Gitbooks page](https://docs.angr.io/ "angr"). A complete [API reference](https://api.angr.io/ "angr API") is also available.
 
-You can use angr from a Python REPL - such as iPython - or script your approaches. Although angr has a bit of a steep learning curve, we do recommend using it when you want to brute force your way to a given state of an executable. Please see the "[Symbolic Execution](0x05c-Reverse-Engineering-and-Tampering.md#symbolic-execution "Symbolic Execution")" section of the "Reverse Engineering and Tampering" chapter as a great example on how this can work.
+You can use angr from a Python REPL - such as iPython - or script your approaches. Although angr has a bit of a steep learning curve, we do recommend using it when you want to brute force your way to a given state of an executable. Please see the "[Symbolic Execution](../../techniques/android/MASTG-TECH-0037.md "Symbolic Execution")" section of the "Reverse Engineering and Tampering" chapter as a great example on how this can work.
```

Comment:
```suggestion
You can use angr from a Python REPL - such as iPython - or script your approaches. Although angr has a bit of a steep learning curve, we do recommend using it when you want to brute force your way to a given state of an executable. Please refer to ["Symbolic Execution"](../../techniques/android/MASTG-TECH-0037.md "Symbolic Execution") as a great example on how this can work.
```